### PR TITLE
Update issues template

### PR DESCRIPTION
**Pull request by lucferbux**
Updated issue template with the new `YAML` format to include a more form-oriented bug and feature request panel.

## Testing Instructions
1. Go to [this issue panel](https://github.com/lucferbux/issue_template_mock/issues/new/choose) where I created a mockup version of this PR.
2. Select either *Bug Report* of *Feature Request*.
3. Check the new form.

## Screenshots

Screenshot 2022-07-07 at 16 31 55
Screenshot 2022-07-04 at 10 35 13

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      label: Workaround (if any)
+      description: Any manual steps that allow you to resolve the issue
+      placeholder: Tell us the steps you followed to resolve the issue!
+    validations:
+      required: false
+  - type: dropdown
+    id: openshift-version
+    attributes:
+      label: OpenShift Version
+      description: What version of OpenShift are you running?
```
Comment (maroroman) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:62]: Would it be possible to add a "other" field here if it is none of the above?

Comment (lucferbux) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:62]: mmm, there's no option to have a multi-select OR other value, we can have a multi-select, one option is "other" and a next field for a optional input, what do you think @dlabaj @cfchase ?

**Comment by LaVLaS** [comment]
This looks really good. I would recommend that we don't ask for "Contact Details" since the submitter's github username is the issue owner and we would not want anyone to feel like they need to make their email address public

**Comment by lucferbux** [comment]
> This looks really good. I would recommend that we don't ask for "Contact Details" since the submitter's github username is the issue owner and we would not want anyone to feel like they need to make their email address public

Mmmm great point, I can add that that field as optional at the end of the form and just rephrase it as "If you wanna be contacted by mail, please add your contact information" or something like that, what do you thing @LaVLaS ?

**Comment by LaVLaS** [comment]
> Mmmm great point, I can add that that field as optional at the end of the form and just rephrase it as "If you wanna be contacted by mail, please add your contact information" or something like that, what do you thing @LaVLaS ?

IMO, we should restrict the discussion to public channels with the default being the GitHub issue and ODH Slack for real time discussions

**Comment by lucferbux** [comment]
> > > Mmmm great point, I can add that that field as optional at the end of the form and just rephrase it as "If you wanna be contacted by mail, please add your contact information" or something like that, what do you thing @LaVLaS ?
> > 
> > 
> > IMO, we should restrict the discussion to public channels with the default being the GitHub issue and ODH Slack for real time discussions
> 
> Yes, I think you're right, I will remove it!

@LaVLaS Done!

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  - dlabaj
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request! Please, fill this form to help us improve the project.
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
```
Comment (DaoDaoNoCode) [inline .github/ISSUE_TEMPLATE/feature_request.yml:17]: Can you also update this?

**Review by dlabaj — APPROVED** [review]
Looks really good.  Thanks for creating this.

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (lucferbux) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: Yes sorry!

**Review by DaoDaoNoCode — APPROVED** [review]
lgtm

**Pull request merged**